### PR TITLE
#784 multiple post arguments in TkStart

### PR DESCRIPTION
**Pull request by dmzaytsev**
For #784 
* TkStart now allows to use multiple post arguments
* added the unit test

### CODE REVIEW

**Comment by karato** [comment]
@dmzaytsev Thanks, let me find someone who can review this pull request

**Comment by karato** [comment]
@pinaf do this review

File: netbout-web/src/test/java/com/netbout/rest/bout/TkStartTest.java [hunk 1]
```diff
@@ -189,4 +190,53 @@ public void preventManyRenameParameters() throws Exception {
             )
         );
     }
+
+    /**
+     * TkStart can handle multiply post parameter.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void handlesMultiplyPost() throws Exception {
```
Comment (pinaf) [inline netbout-web/src/test/java/com/netbout/rest/bout/TkStartTest.java]: @dmzaytsev `handlesMultiplePosts`

File: netbout-web/src/test/java/com/netbout/rest/bout/TkStartTest.java [hunk 2]
```diff
@@ -189,4 +190,53 @@ public void preventManyRenameParameters() throws Exception {
             )
         );
     }
+
+    /**
+     * TkStart can handle multiply post parameter.
```
Comment (pinaf) [inline netbout-web/src/test/java/com/netbout/rest/bout/TkStartTest.java]: @dmzaytsev `can handle multiple post parameters.`

File: netbout-web/src/test/java/com/netbout/rest/bout/TkStartTest.java [hunk 3]
```diff
@@ -189,4 +190,53 @@ public void preventManyRenameParameters() throws Exception {
...
+            "message2",
+        };
+        final String head = new RsPrint(
+            new TkForward(new TkStart(base)).act(
+                new RqWithAuth(
+                    urn,
+                    new RqFake(
+                        RqMethod.GET,
+                        String.format(
+                            "/start?post=%s&post=%s&invite=%s",
```
Comment (pinaf) [inline netbout-web/src/test/java/com/netbout/rest/bout/TkStartTest.java]: @dmzaytsev question: do we need `invite`?

**Comment by pinaf** [comment]
@dmzaytsev nice. 2 minor comments and 1 question.

File: netbout-web/src/test/java/com/netbout/rest/bout/TkStartTest.java (hunk 3 above)
Comment (dmzaytsev) [inline netbout-web/src/test/java/com/netbout/rest/bout/TkStartTest.java]: @pinaf  you're right, I will remove it

**Comment by dmzaytsev** [comment]
@pinaf fixed, take a look please

**Comment by pinaf** [comment]
@rultor merge

**Comment by pinaf** [comment]
@dmzaytsev excellent

**Comment by karato** [comment]
@elenavolokhova please, review this ticket for compliance with our [QA rules](http://www.teamed.io/qa.html)

**Comment by karato** [comment]
@rultor please deploy

**Comment by elenavolokhova** [comment]
@pinaf 
According to our [quality rules](http://www.teamed.io/qa.html):
 > Comments were mostly about design problems, not cosmetic issues.
 
 There were mostly cosmetic issues found during review. Try to pay attention to bigger design problems and the overall quality of the code.
Please confirm that this new rule is clear.

**Comment by pinaf** [comment]
@elenavolokhova I confirm

**Comment by elenavolokhova** [comment]
@alex-palevsky Quality is acceptable here.

**Comment by dmzaytsev** [comment]
@elenavolokhova same here :)
@karato is the PM on this project

**Comment by elenavolokhova** [comment]
@karato Quality is acceptable here.

**Comment by elenavolokhova** [comment]
@dmzaytsev Sorry... I should not work so late at night... Thanks!

**Comment by dmzaytsev** [comment]
@elenavolokhova no problem

**Comment by karato** [comment]
> @karato Quality is acceptable here.

@elenavolokhova thanks a lot, next time everybody should try to make it better

**Comment by karato** [comment]
@pinaf I added 10 mins to @elenavolokhova (for QA review) in transaction 65379614. thanks, paid, **19 mins** to your account, payment ID is `AP-4LX2317047344551Y`, task took 39 hours and 33 mins to complete. review comments (c=4) added as a bonus. +19 added to your rating, current score is: [+3868](http://www.netbout.com/b/35128?open=rating)

**Pull request merged**